bug/op_comment_and_close.go

```diff
@@ -0,0 +1,61 @@
+package bug
+
+import (
+	"github.com/MichaelMure/git-bug/entity"
+	"github.com/MichaelMure/git-bug/identity"
+	"github.com/MichaelMure/git-bug/repository"
+)
+
+var _ Operation = &CommentAndCloseOperation{}
+
+type CommentAndCloseOperation struct {
```

Comment:
Hold on, hold on :)

You don't need a new operation for that, just to trigger both `AddComment` and `Close` in the implementation of the graphQL mutation.

Reply:
I'm at a dead end.
1. I updated [mutations.graphql](https://github.com/MichaelMure/git-bug/blob/5f79e19e0e7211170e0d43924d27692dee2ad1c0/api/graphql/schema/mutations.graphql#L144-L162) and add the Operation type to [operations.graphql](https://github.com/MichaelMure/git-bug/blob/5f79e19e0e7211170e0d43924d27692dee2ad1c0/api/graphql/schema/operations.graphql#L79-L86). But running `go generate` will then result into following error: 
```merging type systems failed: unable to bind to interface: github.com/MichaelMure/git-bug/api/graphql/models.CommentAndCloseBugOperation does not satisfy the interface github.com/MichaelMure/git-bug/bug.Operation```
2. Fix above error by adding the new operation type to graphql-models in [gqlgen.yml](https://github.com/MichaelMure/git-bug/blob/5f79e19e0e7211170e0d43924d27692dee2ad1c0/api/graphql/gqlgen.yml#L45-L46). Running `go generate` again will result in ```modelgen: unable to find type github.com/MichaelMure/git-bug/bug.CommentAndCloseBugOperation
exit status 3```

But does above error not require me to implement this new operation?

Reply:
The root cause of your issue is that you modeled the graphQL mutation the same way as the others (good idea, but that made you fall in the trap) with a single returned `operation`. Later, you had to define this operation, which led to writing an actual operation.

Instead, you should consider that this mutation apply two pre-existing operation and ... return both. Then the code naturally devolve into not creating a new operation.

I pushed the code for that. I haven't tested it but it might be ok anyway. Let me know if that solve the issue for you.

Reply:
There is still some cleanup to do at least.